x-pack/plugins/snapshot_restore/public/application/services/documentation/documentation_links.ts

```diff
@@ -72,6 +72,10 @@ class DocumentationLinksService {
   public getCronUrl() {
     return `${this.esDocBasePath}trigger-schedule.html#schedule-cron`;
   }
+
+  public getCreateIndexTemplateForDataStream() {
+    return `${this.esDocBasePath}set-up-a-data-stream.html#create-a-data-stream-template`;
```

Comment:
As mentioned above, I would link to the "Restore a snapshot" docs.
```suggestion
    return `${this.esDocBasePath}snapshots-restore-snapshot.html`;
```